CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.1.0 (2020-08-10)
+
+- Added the ability for the phone field component to have its country value set programatically. [#51](https://github.com/blackbaud/skyux-phone-field/pull/51)
```

Comment:
I'd tweak this to: 
```Added the ability to programatically set the phone field component's country value.```

No worries if you don't think it's worth the effort though. I'll go ahead and approve the PR and leave it up to you.

Reply:
Done